pages/windows/slmgr.md

```diff
@@ -0,0 +1,37 @@
+# slmgr
+
+> Install, activate, and manage Windows licenses.
+> This command may override, deactivate, and/or remove your current Windows license. Please proceed with caution.
+> More information: <https://docs.microsoft.com/windows-server/get-started/activation-slmgr-vbs-options>.
+
+- [D]isplay the current Windows [l]icense [i]nformation:
+
+`slmgr /dli`
+
+- [D]isplays the ins[t]allation [I]D for the current device. Useful for offline license activation:
```

Comment:
```suggestion
- Displays the ins[t]allation ID for the current device. Useful for offline license activation:
```

`/dti` is not a single letter option to highlight it's first letter in the description. The same applies for other similar comments.

Reply:
While it is a little ugly here, we could also use `[i]D`.